libnm-core/nm-setting-wireless.c

```diff
@@ -885,6 +889,34 @@ verify (NMSetting *setting, NMConnection *connection, GError **error)
 		return FALSE;
 	}
 
+	if (   NM_FLAGS_ANY (priv->wowl, NM_SETTING_WIRELESS_WAKE_ON_WLAN_EXCLUSIVE_FLAGS)
+	    && !nm_utils_is_power_of_two (priv->wowl)) {
```

Comment:
should also regject large flags `!NM_FLAGS_ANY (priv->wowl, NM_SETTING_WIRELESS_WAKE_ON_WLAN_ALL)`.